components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import AccordionItem from '@/components/Accordion/AccordionItem';
+
+import { AccordionProps } from './Accordion.types';
+
+const Accordion: React.FC<AccordionProps> = ({ items }) => {
+  const [openKey, setOpenKey] = React.useState<string | null>(null);
+
+  const updateUrlHash = (key: string | null) => {
+    if (key === null) {
+      window.location.hash = '';
+      return;
+    }
+    window.location.hash = key;
+  };
+
+  const onToggle = (key: string | null) => {
+    if (openKey === key) key = null;
```

Comment:
Keyboard keys should be specified as default like "Enter or Space". Don't need to make it extendable.